core/src/avm2/globals/flash/net/url_loader.rs

```diff
@@ -50,11 +50,22 @@ fn spawn_fetch<'gc>(
         .get_property(&QName::dynamic_name("url").into(), activation)?
         .coerce_to_string(activation)?;
 
+    let method_str = url_request
+        .get_property(&QName::dynamic_name("method").into(), activation)?
```

Comment:
Note for the future: you can use `&Multiname::public("method")` here, no need to go through the `QName`. I want to do a pass converting them everywhere at some point.